### DISCUSSION TITLE
v0.70.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#5508] [Announcements]
### Editor
* Allow embeds to be used inside tables in https://github.com/outline/outline/pull/5315
* add syntax highlighting for jsx and tsx by @Intevel in https://github.com/outline/outline/pull/5330
* Commenting outside of edit mode is now possible when "seamless editing" is disabled. in https://github.com/outline/outline/pull/5422
* fix: Cursor jump on long title edit on Firefox in https://github.com/outline/outline/pull/5449
* fix: Keyboard navigation around inline code marks is now more natural https://github.com/outline/outline/pull/5477
* fix: Comment marks can no longer be copy/pasted between documents in https://github.com/outline/outline/pull/5507

### Platform

* In-app notifications in https://github.com/outline/outline/pull/5354
* The admin role can now be given to users per-collection in https://github.com/outline/outline/pull/5273
* Copy link in comments now includes port if non-standard by @amplitudesxd in https://github.com/outline/outline/pull/5276
* Various sidebar fixes #5278 in https://github.com/outline/outline/pull/5278
* Allow passing subdomain as `null` by @apoorv-mishra in https://github.com/outline/outline/pull/5289
* fix: Refactor attachment downloads during export to use promises #5294 in https://github.com/outline/outline/pull/5294
* Document history sidebar now always allows viewing diff with live document https://github.com/outline/outline/pull/5399
* Add OCI image source label by @MarioNoll in https://github.com/outline/outline/pull/5338
* Dependency bumps in https://github.com/outline/outline/pull/5342
* Update providerId to fallback to id in the absence of a sub field. by @DandrewsDev in https://github.com/outline/outline/pull/5343
* Use `umzug` to autorun migrations by @apoorv-mishra in https://github.com/outline/outline/pull/5281
* Improve reliability by retrying failed imports in https://github.com/outline/outline/pull/5408
* Init app language with DEFAULT_LANGUAGE by @christianrr in https://github.com/outline/outline/pull/5445
* fix: Import max length error not correctly communicated on import in https://github.com/outline/outline/pull/5434
* Add option to not include attachments in exported data in https://github.com/outline/outline/pull/5463
* Remove temporary files after processing by @soulchild in https://github.com/outline/outline/pull/5456
* Improve document revision creation in https://github.com/outline/outline/pull/5474
* Fixed some bugs caused by mishandled promises in https://github.com/outline/outline/pull/5506

## New Contributors
* @amplitudesxd made their first contribution in https://github.com/outline/outline/pull/5276
* @chr4 made their first contribution in https://github.com/outline/outline/pull/5302
* @rohansx made their first contribution in https://github.com/outline/outline/pull/5288
* @libnewton made their first contribution in https://github.com/outline/outline/pull/5313
* @Intevel made their first contribution in https://github.com/outline/outline/pull/5330
* @MarioNoll made their first contribution in https://github.com/outline/outline/pull/5338
* @DandrewsDev made their first contribution in https://github.com/outline/outline/pull/5343
* @christianrr made their first contribution in https://github.com/outline/outline/pull/5445
* @soulchild made their first contribution in https://github.com/outline/outline/pull/5456

**Full Changelog**: https://github.com/outline/outline/compare/v0.69.2...v0.70.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.70.0'>v0.70.0</a>.</em>

**Comment by bmm-alc** [comment]
Thanks for the work and the release
